adafruit_displayio_layout/widgets/flip_input.py

```diff
@@ -508,7 +523,9 @@ def value(self):
         return self._value
 
     @value.setter
-    def value(self, new_value):  # Set the value based on the index or on the string.
+    def value(
+        self, new_value: int | str
```

Comment:
I don't know if I've ever seen the pipe syntax before. Is that different than `Union` functionally? or just a different way to write the same thing?

Reply:
@foamyguy I mentioned it in my previous response but it's syntax that equates to using Union.  It is not available in Python 3.8 however.